unit_1d_hw.py

```diff
@@ -7,6 +7,128 @@
 '''
 import random # We will talk about imports later :)
 
+'''
+DEV NOTES:
+
+For Unit 1D They learn te below:
+
+- Functions 
+    - Creating a function
+    - Returning a value (including void)
+    - Creating parameters for function
+    - Default arguments
+    - Multiple returns
+
+- Classes
+    - calling .append on a list (may need to use a fixture for this)
+    - Creating an object 
+    - Creating the init function
+    - Creating more functions in the same class
+
+
+Using built in functions
+- Range
+- Possibly others
+
+'''
+
+#### Create a Function Below! ####
+'''
+Create a function called function_1(), have it return None
+
+Hint: Syntax to create a function:
+
+def <function_name>():
+    return <some value>
+
+'''
+
+##################################
+
+#### Create a Function Below ####
+'''
+Create a function called function_return_1() that returns the integer 5
+
+'''
+
+##################################
+
+#### Create a Function Below ####
+'''
+Create a function called function_return_1() that returns the string "I am a string!"
+
+'''
+
+##################################
+
+#### Create a Function Below ####
+'''
+Create a function called add_one, that takes in a parameter x, adds 1 then returns that value
```

Comment:
I noticed that from here on out you say the names of the functions without `()`. Doesn't really matter (I hope they'll be able to figure it out), but wanted to note it in case you wanted to be consistent throughout.

Reply:
Yes, good catch. I am going to have it all have the () to be more clear. I think this will get a bit confusing later, but they will figure it out.